backend/btrixcloud/crawlconfigs.py

```diff
@@ -155,6 +155,13 @@ def sanitize(self, string=""):
         """sanitize string for use in wacz filename"""
         return self._file_rx.sub("-", string.lower())
 
+    async def get_profile_filename(
+        self, profileid: Union[UUID, EmptyStr, None], org: Organization
```

Comment:
Not sure the `EmptyStr` is necessary here? Unless mypy caught something I'm not seeing, which is possible.

Reply:
yep, you're right, it is needed in the more general lookup as that can include empty string for updated, however the crawlconfig.profileid is an Optional[UUID] so that should be fine.